cachi2/core/package_managers/gomod.py

```diff
@@ -582,7 +582,37 @@ def _resolve_gomod(
         # Make Go ignore the vendor dir even if there is one
         go_list.extend(["-mod", "readonly"])
 
-    main_module_name = _run_gomod_cmd([*go_list, "-m"], run_params).rstrip()
+    go_list_modules = _run_gomod_cmd([*go_list, "-m", "-json"], run_params).rstrip()
+    module_list = list(load_json_stream(go_list_modules))
+    raw_main_module = module_list.pop(0)
```

Comment:
I'm curious,  why is this named a raw module? How is it different from any other module in the list?

Reply:
There could probably be some better name. Later on there is `main_module` which is `ParsedModule()`, but these data that are `dict` type correspond to main module data.